Automattic/jetpack
# Actions: add new action to run k6 tests against staging sites

## Proposed changes:

Add some k6 tests to generate traffic on predefined staging/test sites.

### Other information:


## Jetpack product discussion

Internal PT: p9dueE-6Re-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

The backend tests will require the proper secrets, but you can see an example action run here: https://github.com/samiff/test/actions/runs/5225021803/jobs/9433951842

## Todo:

- [x] Add new secrets to repo.
- [x] Add Slack alerting for failures (also consider this for the action that updates the staging sites).
